desktop/ui/src/test/java/org/datacleaner/job/runner/RowProcessingConsumerSorterTest.java

```diff
@@ -76,7 +77,7 @@ public void testCreateProcessOrderedConsumerListWithMergedOutcomes() throws Exce
         fjb1.setName("fjb1");
 
         // 2: trim (depends on filter)
-        TransformerComponentBuilder<TransformerMock> tjb1 = ajb.addTransformer(TransformerMock.class);
+        TransformerComponentBuilder<MockTransformer> tjb1 = ajb.addTransformer(MockTransformer.class);
```

Comment:
No need to do it now, but should the `TransformerMock` class die completely, or does it serve a purpose in the rest of `RowProcessingConsumerSorterTest`? It seems to be the only class that doesn't use the normal `MockTransformer`